crates/uv-toolchain/src/downloads.rs

```diff
@@ -0,0 +1,55 @@
+#[derive(Debug, PartialEq)]
+pub struct PythonDownloadMetadata {
+    name: ImplementationName,
+    arch: Arch,
+    os: Os,
+    libc: Libc,
+    major: i64,
+    minor: i64,
+    patch: i64,
+    url: &'static str,
+    sha256: Option<&'static str>,
+}
+
+#[derive(Debug, PartialEq)]
+pub(crate) enum Arch {
+    Arm64,
+    I686,
+    Ppc64Le,
+    S390X,
+    Windows,
+    X86_64,
+}
+
+#[derive(Debug, PartialEq)]
+pub(crate) enum Libc {
+    Gnu,
+    Musl,
+    None,
+}
+
+#[derive(Debug, PartialEq)]
+pub(crate) enum ImplementationName {
+    Cpython,
+}
+
+#[derive(Debug, PartialEq)]
+pub(crate) enum Os {
+    Darwin,
+    Linux,
+    Shared,
+    Windows,
+}
+include!("python_versions.inc");
```

Comment:
Is there a reason this is called `.inc` and not `.rs`?

Reply:
Coping Rye, but it seems clearer that it's generated.